package.json

```diff
@@ -6,7 +6,7 @@
     "backbone.paginator": "~2.0.3",
     "coffee-script": "1.6.1",
     "edx-pattern-library": "0.18.1",
-    "edx-ui-toolkit": "1.5.1",
+    "edx-ui-toolkit": "1.5.3",
```

Comment:
Does this need a corresponding `package-lock.json` update?

It looks like [the current version of `edx-ui-toolkit` in `package-lock.json` in `master` right now is `1.5.2`](https://github.com/edx/edx-platform/blob/master/package-lock.json#L2632-L2647).

![image](https://user-images.githubusercontent.com/8136030/35071212-ea69f032-fbad-11e7-8bfe-4093bc37a1c3.png)

Reply:
Geez, I can't read - this was closed and my comment is irrelevant. Somehow this made it into my GitHub notifications. Please ignore.